webpack/webpack.config.js

```diff
@@ -10,22 +10,10 @@ module.exports = {
         library: 'carto',
         libraryTarget: 'umd'
     },
-    devtool: 'sourcemap',
+    devtool: 'source-map',
```

Comment:
This seems to be the correct value, as described in https://webpack.js.org/configuration/devtool